app/src/main/java/org/mifos/mobile/ui/activities/LoginActivity.kt

```diff
@@ -57,6 +55,24 @@ class LoginActivity : BaseActivity(), LoginView {
         setContentView(R.layout.activity_login)
         ButterKnife.bind(this)
         loginPresenter?.attachView(this)
+        dismissSoftKeyboardOnBkgTap(findViewById(R.id.nsv_bkg))
```

Comment:
hmmm... as you you are binding the view with ButterKnife, you could access the view directly by its id, I think there is no need for `findViewById`, agree?

Reply:
Yes I agree. Made the changes.